src/Umbraco.Web.Common/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -261,6 +260,7 @@ private static IUmbracoBuilder AddHttpClients(this IUmbracoBuilder builder)
                 ServerCertificateCustomValidationCallback =
                     HttpClientHandler.DangerousAcceptAnyServerCertificateValidator,
             });
+        builder.Services.AddHttpClient(Constants.HttpClients.WebhookFiring);
```

Comment:
Should this set a default User-Agent (e.g. `Umbraco/13.0.0`), like other tools also do: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#library_and_net_tool_ua_strings?